src/bundle/Resources/public/js/alloyeditor/src/buttons/parts/ez-discover-content.js

```diff
@@ -0,0 +1,66 @@
+import React, { createRef } from 'react';
+import PropTypes from 'prop-types';
+import AlloyEditor from 'alloyeditor';
+import EzEmbedDiscoverContentButton from '../../base/ez-embeddiscovercontent';
+
+export default class EzDiscoverContent extends EzEmbedDiscoverContentButton {
+    constructor(props) {
+        super(props);
+
+        this.originalSetUIHidden = null;
+        this.buttonRef = createRef(null);
+    }
+
+    confirmSelection() {
+        const { confirmSelectedItems, editor } = this.props;
+
+        confirmSelectedItems(...arguments);
+
+        ReactDOM.unmountComponentAtNode(document.querySelector('#react-udw'));
+
+        editor._mainUI._setUIHidden = this.originalSetUIHidden;
+
+        this.buttonRef.current.focus();
+    }
+
+    cancelHandler(udwContainer) {
+        super.cancelHandler(udwContainer);
+
+        this.props.editor._mainUI._setUIHidden = this.originalSetUIHidden;
+
+        this.buttonRef.current.focus();
+    }
+
+    chooseContent() {
+        const { editor } = this.props;
+
+        this.originalSetUIHidden = editor._mainUI._setUIHidden;
+
+        editor._mainUI._setUIHidden = () => {};
+
+        super.chooseContent();
+    }
+
+    render() {
+        const buttonLabel = Translator.trans(/*@Desc("Select Content")*/ 'discover_content.button.label', {}, 'alloy_editor');
+
+        return (
+            <div>
+                <button className="btn btn-secondary ez-btn-ae" ref={this.buttonRef} onClick={this.chooseContent.bind(this)}>
```

Comment:
Why not bind it in constructor? As I saw, it was the way for all other places, so it's little incosistent.

(Also, by binding it here, you call bind with each rerender, which is unnecessary)

Reply:
Probably during development, I bind some value here but removed it so that's why this stayed here.